### DISCUSSION TITLE
How to create index over collection name with forward slashes '/'

### DISCUSSION THREAD

**Discussion by mirecg** [ravendb/ravendb#13144] [Q&A] [answered]
I've created collection with name using forward slash '/' but it seems that while I'm able to run Query over it, I'm not able to create index on it in Studio.
For example:
Collection name = "Windows/Closed"
Query: OK
```
from 'Windows/Closed'
```

Index: fails to compile
```
from w in docs.Windows/Closed
select new
{
    LastUpdated = w.LastUpdated
}
```
I don't know how to escape the '/'  or what is the syntax for such collection naming.

**Comment by grisha-kotler** [comment, marked as answer]
```
from w in docs["Windows/Closed"]
select new
{
    LastUpdated = w.LastUpdated
}
```
